db/fixtures/chargeback_rates.yml

```diff
@@ -12,6 +12,7 @@
       :per_time: daily
       :per_unit: megahertz
       :measure: Hz Units
+      :type_currency: Dollars
```

Comment:
Note, I found my answer... type_currency is only used as a lookup to find the currency, it's not a column in the db.